praw/const.py

```diff
@@ -90,6 +90,7 @@
     'my_multireddits':        'api/multi/mine/',
     'my_subreddits':          'subreddits/mine/subscriber/',
     'marknsfw':               'api/marknsfw/',
+    'modmail_conversation':   '/api/mod/conversations/{id}',
```

Comment:
Does it still work without the leading `/`? If so, that would be preferred for consistency.